src/MemoryGame/Helpers.java

```diff
@@ -0,0 +1,17 @@
+package MemoryGame;
+
+import java.util.Arrays;
+
+public class Helpers {
+    public static void printAndClearAfterDelay(String message, long delayInMillis) {
+        System.out.println(message);
+        try {
+            Thread.sleep(delayInMillis);
+        } catch (InterruptedException e) {
+            System.out.println(Arrays.toString(e.getStackTrace()));
+        }
+        for (int i = 0; i < 50; i++) {
```

Comment:
This is an interesting idea, although wouldn't the user technically be able to cheat by scrolling up lol

Reply:
Haha hell yea, its so easy to cheat. I tried looking up if there was a way that we could write to the console and delete it but idek if thats possible in Java. Thats why I want to learn Swing or Javafx. I might study it this weekend.